CODIGO/BabelUI.bas

```diff
@@ -0,0 +1,391 @@
+Attribute VB_Name = "BabelUI"
+Option Explicit
+
+Const CreateCharMap = 782
+Const CreateCharMapX = 25
+Const CreateCharMapY = 35
+Private Type LOGINDATA
+    User As Long
+    UserLen As Long
+    password As Long
+    PasswordLen As Long
+    storeCredentials As Long
+End Type
+
+Private Type NewAccountData
+    User As Long
+    UserLen As Long
+    password As Long
+    PasswordLen As Long
+    Name As Long
+    NameLen As Long
+    Surname As Long
+    SurnameLen As Long
+End Type
+
+Private Type NEWCHARACTERDATA
+    name As Long
+    NameLen As Long
+    Gender As Long
+    Race As Long
+    Class As Long
+    Head As Long
+    City As Long
+End Type
+
+Private Type SINGLESTRINGPARAM
+    Ptr As Long
+    Len As Long
+End Type
+
+Private Type DOUBLESTRINGPARAM
+    FirstPtr As Long
+    FirstLen As Long
+    SecondPtr As Long
+    SecondLen As Long
+End Type
+
+Private Type TRIPLESTRINGPARAM
+    FirstPtr As Long
+    FirstLen As Long
+    SecondPtr As Long
+    SecondLen As Long
+    ThirdPtr As Long
+    ThirdLen As Long
+End Type
+
+Private Type BABELSETTINGS
+    Width As Long
+    Height As Long
+    Compresed As Long
+    EnableDebug As Long
+End Type
+
+Private ServerEnvironment As String
+
+Public Declare Function InitializeBabel Lib "BabelUI.dll" Alias "_InitializeBabel@4" (ByRef Settings As BABELSETTINGS) As Boolean
```

Comment:
Why `@4` in `_InitializeBabel@4` ? the same for all the other public functions in the dll, it should not be necessary to use `@x`

Reply:
I was not able to configure a different calling convention that works with vb6 not sure why

Reply:
Ok we need to fix it, I have this solved in `steam_vb.dll` its a matter of using the same configuration

Reply:
lets have a call to look at this then I tried multiple settings but when I exposed other calling conventions VB6 could not find the simbols